### ISSUE TITLE
 pre-industrial: Use executables created by Spack

### ISSUE THREAD

**Issue by penguian** [ACCESS-NRI/access-esm1.5-configs#16]
For the `pre-industrial` configuration, use executables created via a Spack build such as https://github.com/ACCESS-NRI/spack-packages/blob/main/packages/access-esm1p5/package.py

**Comment by aidanheerdegen** [comment]
You can use the [pre-release binaries deployed on to `gadi`](https://github.com/ACCESS-NRI/ACCESS-ESM1.5/pull/5#issuecomment-2144408629)

```
module use /g/data/vk83/prerelease/modules/access-models/ && module load access-esm1p5/pr5-17
```

then find the locations of the executables:
```
$ which cice_access_360x300_12x1_12p.exe
/g/data/vk83/prerelease/apps/spack/0.21/restricted/ukmo/release/linux-rocky8-x86_64/intel-19.0.5.281/cice4-git.2024.05.21_access-esm1.5-qogjlyco4cjbdilrx3hsqc2vbvxv4lwk/bin/cice_access_360x300_12x1_12p.exe
$ which um_hg3.exe
/g/data/vk83/prerelease/apps/spack/0.21/restricted/ukmo/release/linux-rocky8-x86_64/intel-19.0.5.281/um7-git.2024.05.21_7.3-qkwgouyf37lf5vnngpoh6zsmsj4hg55d/bin/um_hg3.exe
$ which fms_ACCESS-CM.x
/g/data/vk83/prerelease/apps/spack/0.21/restricted/ukmo/release/linux-rocky8-x86_64/intel-19.0.5.281/mom5-git.access-esm1.5_2024.05.24_access-esm1.5-ttg4y4yt3ddzhjywf5yfiicibk6xkx22/bin/fms_ACCESS-CM.x
```
You can then put those paths into the `config.yaml` and it should "just work".

Note that `payu` [now supports using the module system to change binary versions](https://github.com/ACCESS-NRI/model-config-tests/issues/10) thanks to @jo-basevi.

This version of `payu` is not currently deployed on `gadi` though there are plans to do so. 

There is nothing to stop you changing the executable names to match those above and installing a local copy of `payu` so you can do something like this:
```yaml
modules:
  use:
      - /g/data/vk83/prerelease/modules/access-models
  load:
      - access-esm1p5/pr5-17
```
This will be how the configurations will be for release, and I apologise this feature isn't available in a deployed version of `payu` yet.

**Comment by penguian** [comment]
The initial attempt to use the `vk83` executables as above fails with a segmentation violation in all 180 MOM5 ranks, as per https://github.com/ACCESS-NRI/ACCESS-ESM1.5/issues/6

**Comment by aidanheerdegen** [comment]
Note that
```yaml
modules:
  use:
      - /g/data/vk83/prerelease/modules/access-models
  load:
      - access-esm1p5/pr5-17
```
should now work with the `payu/dev` version that is now available on `Gadi`

**Comment by penguian** [comment]
Closed by PR #26
